holmes/plugins/toolsets/skills/skills_fetcher.py

```
@@ -139,6 +153,38 @@ def _format_skill_result(self, skill: Skill, params: dict) -> StructuredToolResu
+    ) -> Optional[StructuredToolResult]:
+        """Fetch a personal skill scoped to this end user.
+
+        Returns None when the id is not one of that user's personal skills, so the caller
+        can fall through to the global lookup. Errors are also returned as None rather than
+        surfaced, because a miss here is the normal case for a global skill id.
+        """
+        if not self._dal:
+            return None
+        try:
+            skill_content = self._dal.get_personal_skill_content(skill_id, user_id)
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)